rtf_vis_tool/src/App.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import {Canvas, extend} from "react-three-fiber";
+import {OrbitControls} from 'three/examples/jsm/controls/OrbitControls';
+import './App.css';
+
+extend({OrbitControls})
+
+const App = (props) => {
+  return (
+    <p>boilerplate App.js body.</p>
+  )
+}
+
+export default App;
```

Comment:
just a simple App.js React component. This component is imported in `index.js`

Reply:
thanks. do you mind adding a brief inline comment explaining what OrbitControls and Canvas are used for?